bucket/mpc-qt.json

```
@@ -0,0 +1,20 @@
+{
+    "version": "22.02",
+    "description": "A clone of Media Player Classic reimplemented in Qt.",
+    "homepage": "https://github.com/mpc-qt/mpc-qt",
+    "license": "GPL-2.0-only",
+    "suggest": "yt-dlp",
```

```suggestion
    "suggest": {
        "yt-dlp": "yt-dlp"
    },
```
**suggest** is designed to be used like:
```
"suggest": {
    "Java Runtime Environment": "java/openjdk18"
}
```